my-app/src/App.js

```diff
@@ -0,0 +1,25 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.js</code> and save to reload.
+        </p>
+        <a
```

Comment:
*THIS IS AN EXAMPLE*
Let's create a Link component since we are using links quite often in this application

Reply:
*NOTE FOR DEVS*
You can have a small conversation here in regards to the comment above. If a longer conversation is needed, move it to Discord